tools/spectral/ipa/rulesets/functions/eachPathAlternatesBetweenResourceNameAndPathParam.js

```diff
@@ -0,0 +1,34 @@
+import { isPathParam } from './utils/pathUtils';
+
+const ERROR_MESSAGE = 'API paths must alternate between resource name and path params.';
+const ERROR_RESULT = [{ message: ERROR_MESSAGE }];
+const AUTH_PREFIX = '/api/atlas/v2';
+const UNAUTH_PREFIX = '/api/atlas/v2/unauth';
+
+const getPrefix = (path) => {
+  if (path.includes(UNAUTH_PREFIX)) return UNAUTH_PREFIX;
+  if (path.includes(AUTH_PREFIX)) return AUTH_PREFIX;
+  return null;
+};
+
+const validatePathStructure = (elements) => {
+  return elements.every((element, index) => {
+    const isEvenIndex = index % 2 === 0;
+    return isEvenIndex ? !isPathParam(element) : isPathParam(element);
+  });
+};
+
+// eslint-disable-next-line no-unused-vars
+export default (input, _0, _1) => {
```

Comment:
```suggestion
export default (input) => {
```
I think this still works and you won't need to disable eslint